src/otx/v2/adapters/torch/mmengine/mmpretrain/model.py

```diff
@@ -147,11 +148,15 @@ def get_model(
             if head and hasattr(head, "num_classes"):
                 model["model"]["head"]["num_classes"] = num_classes
 
-    model = get_mmpretrain_model(model, pretrained=pretrained, **kwargs)
+    torch_model = get_mmpretrain_model(model, pretrained=pretrained, **kwargs)
 
-    if channel_last and isinstance(model, torch.nn.Module):
-        model = model.to(memory_format=torch.channels_last)
-    return model
+    source_config = deepcopy(torch_model._config.model)  # noqa: SLF001
```

Comment:
Can you add a comment here why this is needed? This would be for a future reference